src/WasmExecutor.cpp

```diff
@@ -16,8 +16,8 @@
 #include <vector>
 
 #ifdef WITH_V8
-#include "v8.h"
 #include "libplatform/libplatform.h"
+#include "v8.h"
```

Comment:
I have a feeling this order might be important. There was a reason I skipped it last time

Reply:
In general, relying on #include order being important is a code smell -- aside from preprocessor-list-of-stuff hacks, which are a special case and should definitely be called out with no-clang-format.

Reply:
edit: ah, no, you are probably correct here, as v8 is kinda special. will wrap.